docs/process/teams.md

```diff
@@ -0,0 +1,12 @@
+These teams are not set in stone and will likely evolve as we continue to iterate on what makes the most sense for the success of the product.
+
+| Team | Purpose |
+|:--|:--|
+| Analytics | Analyzing the data we use for gaining insight into Desktop and its users |
+| Core | Internal GitHub team that maintains Desktop |
+| Comrades | Avenue for community members to get more involved in the Desktop project |
+| Docs | Responsible for creating documentation Desktop users can consult when needed |
+| Eng | Responsible for implementing Desktop's new features, fixing bugs, and all other developement tasks |
+| Product | Ensures Desktop addresses the needs of its users by prioritizing what to build |
+| Product/Design | Responsible for user experience and design of things implemented in Desktop |
+| QA | Conducts ongoing manual testing of new features to ensure Desktop works reliably for users |
```

Comment:
Similar to above - can we name this group `desktop/quality` or maybe `desktop/quality-assurance`? @meaghanlewis is there a name you prefer here?

Reply:
I'm fine with changing the names, but one thing to note is all current mentions to `desktop/qa` will be broken as a result of the rename.

Reply:
We'll be breaking a few things anyway as part of the rename of `core`. This is fine, as long as we settle on good names and only break things once.